### ISSUE TITLE
How to use Wikitude with React Native

### ISSUE THREAD

**Issue by lehuydev2203** [brave-digital/react-native-wikitude#11]
In function , startAR( architectWorldURL: string, hasGeolocation:boolean, hasImageRecognition:boolean , hasInstantTracking:boolean , wikitudeSDKKey:string ) ", I don't understand paramater '  architectWorldURL ' . What is it ???

**Comment by JanOwiesniak** [comment]
Wikitude allows you to build your own AR Scene with a web based tool called "WIkitude Studio".
This scene is basically a static website which defines your ar components.
It will be loaded by the Wikitude SDK. You can download this scene and host it where ever you like (e.g. AWS / GCP) or pay Wikitude a little extra fee and leave this task to them.

**Comment by lehuydev2203** [comment]
As far as I understand, I have to build a website containing objects, and then with the site address, I will add to the architectWorldURL section.

**Comment by JanOwiesniak** [comment]
This is correct. The static "website" will be build by "Wikitude Studio". There is an "export" link which allows you to download the scene to your system. Then host ist anywhere you like. What works for me was starting a simple web server to serve the website e.g. https://www.npmjs.com/package/http-server and exposing it public via https://ngrok.com. You would then use the ngrok url as your architectWorldURL. This workflow worked great for me. For production you need to find other ways of course but it's good enough for development.

**Comment by nbastoWM** [comment]
I'm also struggling with this problem. I exported a WTO file (I don't see any other form of export), I have a direct link to the file ( https://drive.google.com/uc?export=download&id=1xvcGmp7fGdfR-8K__BaHejxMO8JQ00ud ) but when I run the App in react native with this url it just redirects to the browser to download the file.

**Comment by JanOwiesniak** [comment]
You must provide a url to a html file, not to the WTO file. You can download your scene from "Wikitude Studio" as well. Besides that, you could try a [sample scene](https://github.com/Wikitude/wikitude-sdk-samples) first. If this is working, replace the sample scene with your scene (downloaded from "Wikitude Studio").

**Comment by nbastoWM** [comment]
Thank you! Just found the "Download offline project..." option.

The problem now is :
"Version Conflict
The used Wikitude SDK 7.2.1 is not supported. 
Please update the application."

I tried to start a new project in Wikitude studio with sdk 7 but it only allows to add "object targets" as videos, not images.

**Comment by JanOwiesniak** [comment]
You could try to update the SDK to version 8.1.0. [My pull request was merged some time ago](https://github.com/brave-digital/react-native-wikitude/pull/9). Keep in mind that there seems to be [some problems](https://github.com/brave-digital/react-native-wikitude/issues/8) with the NPM Package (i guess) which results into a missing iOS SDK, so you probably have to link it manually in XCode.

If you have done this, create a new Wikitude Studio Project based on Version 8.1.0 and try to integrate your scene again. If you need any further help, just let me know.

**Comment by nbastoWM** [comment]
Thanks for the help. 
It seems that the aar in npm package is the old sdk. I extracted the last sdk (WikitudeSDK_Android_8-2-0_2018-11-13_11-32-16) and replaced the aar file in wikitudesdk folder. Now there's no version conflict, however as soon as the the camera appears, it keeps showing the  "loading..." with the animated yellow bars. I managed to launch it also in iOS and it also shows the loading bars. (it's been like that for 10 minutes already)

**Comment by JanOwiesniak** [comment]
Are you experiencing the same issues with a [sample scene](https://github.com/Wikitude/wikitude-sdk-samples)?

**Comment by nbastoWM** [comment]
It looks like the problem was with the server I was hosting the project.
Now it's running OK.
Thank you again.

**Comment by zaratejv** [comment]
Hi Brave Team,

I am trying to call a java method within my function via document.location://architectsdk/testMethod.  However, when deploying my application to my emulator, I am currently getting an unknown schema exception..  Am I doing something wrong? Is this feature currently not supported? If so could you please point me in the right direction to get this working?

Thanks!
